### ISSUE TITLE
Dummy gives incorrect result on sympy.sympify

### ISSUE THREAD

**Issue by sidhu1012** [symengine/symengine.py#372]
Original issue https://github.com/sympy/sympy/issues/22141

```
>>> import sympy, symengine
>>> a = symengine.symbols('a', cls = symengine.Dummy)
>>> a
_a
>>> sympy.sympify(a)
__a
```

`sympify` should return a dummy without the additional underscore.

The problem is here-

https://github.com/symengine/symengine.py/blob/5d49ad3b093ddce56ed13568b256dd74b8eae61d/symengine/lib/symengine_wrapper.pyx#L1263-L1265

the Dummy conversion takes the dummy name of symengine type which gives it two underscores

**Comment by certik** [comment]
Awesome. Thanks for the report. It seems that should be easy to fix?

**Comment by sidhu1012** [comment]
Yes , it's an easy to fix issue

The optimal fix would be to ignore the underscore by doing something like

```
return sympy.Dummy(str(self)[1:])
```

**Comment by isuruf** [comment]
It's not. PRs are welcome

**Comment by faze-geek** [comment]
@isuruf 
Could you check if this is fixed !?

Currently on terminal-
```
>>> import sympy, symengine
>>> a = symengine.symbols('a', cls = symengine.Dummy)
>>> a
_a
>>> sympy.sympify(a)
_a

```

**Comment by isuruf** [comment]
Fixed in #375
